src/main/java/fr/insee/publicenemy/api/configuration/swagger/role/DisplayRolesOnSwaggerUI.java

```diff
@@ -0,0 +1,43 @@
+package fr.insee.publicenemy.api.configuration.swagger.role;
```

Comment:
maybe you don't need this. It is used to display roles allowed to use endpoints in swagger UI